tests/test_modeling.py

```diff
@@ -60,19 +61,25 @@ def test_from_save_transformers(self):
             transformers_model = self.trl_model_class.transformers_parent_class.from_pretrained(model_name)
 
             trl_model = self.trl_model_class.from_pretrained(model_name)
-            
+
             with tempfile.TemporaryDirectory() as tmp_dir:
                 trl_model.save_pretrained(tmp_dir)
                 transformers_model_from_save = self.trl_model_class.transformers_parent_class.from_pretrained(tmp_dir)
-            
-            # Check if the weights are the same 
+
+            # Check if the weights are the same
             for key in transformers_model.state_dict():
-                self.assertTrue(torch.allclose(transformers_model_from_save.state_dict()[key], transformers_model.state_dict()[key]))
+                self.assertTrue(
+                    torch.allclose(
+                        transformers_model_from_save.state_dict()[key], transformers_model.state_dict()[key]
+                    )
+                )
+
 
 class VHeadModelTester(BaseModelTester, unittest.TestCase):
```

Comment:
not for this pr, but a small nit that being explicit with e.g. `ValueHeadModelTester` is generally best practice + enables newcomers to easier understand the codebase. 

the same nit applies to docstrings using `v-head` vs `value head` and filenames like `modeling_vhead.py` being better named as `modeling_value_head.py`

Reply:
Addressed in https://github.com/lvwerra/trl/pull/65 !